### PR TITLE
some controls added

### PR DESCRIPTION
**Pull request by broslukasz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,10 @@
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
     "@fontsource/roboto": "^5.0.5",
+    "@mui/icons-material": "^5.14.0",
     "@mui/material": "^5.13.7",
+    "@mui/x-date-pickers": "^6.10.0",
+    "dayjs": "^1.11.9",
```
Comment (DBulinski) [inline package.json]: Do you have strong preferences using dayjs instead of date-fns?

File: package.json [hunk 2]
```diff
@@ -37,9 +40,8 @@
...
   },
   "lint-staged": {
-    "*.{js,css,ts,tsx,jsx}": [
-      "prettier --write",
+    "*.{js,ts,tsx,jsx}": [
       "eslint --fix"
     ]
   }
-}
+}
```
Comment (DBulinski) [inline package.json:47]: missing empty end line, I think it should be configured in your IDE :)

https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file

File: src/App.tsx [hunk 3]
```diff
@@ -11,6 +9,13 @@ import i18n from './i18n';
 import { useTranslation } from 'react-i18next';
 
 import Button from '@mui/material/Button';
+import { AppBar, Toolbar, MenuItem } from '@mui/material';
+import Typography from '@mui/material/Typography';
```
Comment (DBulinski) [inline src/App.tsx:12]: I think it's good to be consistent, either import all components as default exports, or use named exports everywhere :) 
```
import AppBar from '@mui/material/AppBar';
import Toolbar from '@mui/material/Toolbar';
import MenuItem from '@mui/material/MenuItem';


```

File: src/App.tsx [hunk 4]
```diff
@@ -22,7 +27,43 @@ function App() {
...
+            Invoices
+          </Typography>
+
+          <MenuItem>
+            <Typography textAlign="center">New Invoice</Typography>
+          </MenuItem>
+        </Toolbar>
+      </AppBar>
+
+      <InvoiceList></InvoiceList>
```
Comment (DBulinski) [inline src/App.tsx]: Generally, when component doesn't accept children, the convention is to use self closing elements `<InvoicesList /> `

File: src/components/Invoice-list.tsx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  createData('24/23/20', new Date('04/22/2020').toDateString(), new Date('12/6/2021').toDateString(), 838, null),
+  createData('25/23/20', new Date('04/22/2020').toDateString(), new Date('12/6/2021').toDateString(), 838, null),
+];
+
+export default function InvoiceList() {
+  return (
+    // Skąd lepiej importować? Bardziej czy mniej szczegółowo?
+    <Grid container spacing={2} p={4} direction={'column'} mb={0}>
+      <TableContainer component={Paper}>
+        <Table sx={{ minWidth: 650 }} aria-label="simple table">
```
Comment (DBulinski) [inline src/components/Invoice-list.tsx]: I think the table should have a valid label :D

File: src/components/Invoice-list.tsx [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+  createData('21/23/20', new Date('04/22/2020').toDateString(), new Date('12/6/2021').toDateString(), 838, null),
+  createData('22/23/20', new Date('04/22/2020').toDateString(), new Date('12/6/2021').toDateString(), 838, null),
+  createData('23/23/20', new Date('04/22/2020').toDateString(), new Date('12/6/2021').toDateString(), 838, null),
+  createData('24/23/20', new Date('04/22/2020').toDateString(), new Date('12/6/2021').toDateString(), 838, null),
+  createData('25/23/20', new Date('04/22/2020').toDateString(), new Date('12/6/2021').toDateString(), 838, null),
+];
+
+export default function InvoiceList() {
+  return (
+    // Skąd lepiej importować? Bardziej czy mniej szczegółowo?
```
Comment (DBulinski) [inline src/components/Invoice-list.tsx]: Generally it will be slightly more performant, when using default imports, but it's not a big deal. I would rather try to be consistent. You can read more here:
https://mui.com/material-ui/guides/minimizing-bundle-size/#when-and-how-to-use-tree-shaking

File: src/components/Invoice.tsx [hunk 7]
```diff
@@ -0,0 +1,115 @@
+import { Grid, TextField, Button, Typography, Box, IconButton } from '@mui/material';
+import { LocalizationProvider, DatePicker } from '@mui/x-date-pickers';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+
+import SaveIcon from '@mui/icons-material/Save';
+import DeleteIcon from '@mui/icons-material/Delete';
+
+export function Invoice() {
```
Comment (DBulinski) [inline src/components/Invoice.tsx]: Mixed exports, you are using named export here, and default export in `Invoice-list.tsx`

File: src/App.tsx [hunk 8]
```diff
@@ -22,7 +27,43 @@ function App() {
...
+
+      <Invoice></Invoice>
+
+      {/* Pytanie: dlaczego jak daje m={2} dodaje mi poziomego scrolla */}
+      <Grid container spacing={2} p={4} direction={'column'} mb={12}>
+        <Grid item container>
```
Comment (DBulinski) [inline src/App.tsx]: Why is this needed here? I think something like that:
```        
<Grid container spacing={2} p={4} direction={'column'} mb={12}>
            <Grid
                item
                xs={6}
                container
                justifyContent="center"
                alignItems="flex-start"
                gap={2}
                sx={{ height: 'fit-content' }}
            >
                {/* Kiedy to usuwam to pojawia się błąd w konsoli */}
                <Button variant="contained" startIcon={<SaveIcon />}>
                    Send
                </Button>
            </Grid>
        </Grid>
```
Should have the same effect?

File: src/App.tsx [hunk 9]
```diff
@@ -22,7 +27,43 @@ function App() {
...
+            <Typography textAlign="center">New Invoice</Typography>
+          </MenuItem>
+        </Toolbar>
+      </AppBar>
+
+      <InvoiceList></InvoiceList>
+
+      <Invoice></Invoice>
+
+      {/* Pytanie: dlaczego jak daje m={2} dodaje mi poziomego scrolla */}
```
Comment (DBulinski) [inline src/App.tsx]: If you inspect this container you will see that it has `width: calc(100% + [n]px)` where n is your margin. This is top level grid container so by default it expands to the full width, if you add any margin, it will exceed 100% causing horizontal scroll. We can discuss at friday ;)

File: src/App.tsx [hunk 10]
```diff
@@ -22,7 +27,43 @@ function App() {
...
+            item
+            xs={6}
+            container
+            justifyContent="flex-end"
+            alignItems="flex-start"
+            gap={2}
+            sx={{ height: 'fit-content' }}
+          >
+            {/* Kiedy to usuwam to pojawia się błąd w konsoli */}
+            <Button variant="contained" startIcon={<SaveIcon />}>
```
Comment (DBulinski) [inline src/App.tsx]: Can't reproduce the issue, 

- [ ] Todo: discuss at friday ;)

File: src/components/Invoice.tsx [hunk 11]
```diff
@@ -0,0 +1,115 @@
...
+              <TextField id="recipient-bank-account" label="Bank Account" variant="standard" fullWidth />
+            </Box>
+          </Grid>
+          <Grid xs={6} item>
+            <Typography variant="h4" component="p">
+              Sender
+            </Typography>
+
+            <Box mt={6}>
+              {/* Pytanie Dlaczego czasem daje kursor na koniec */}
```
Comment (DBulinski) [inline src/components/Invoice.tsx:74]: Not sure I understand the issue here

- [ ] Discuss at friday

File: src/components/Invoice.tsx [hunk 12]
```diff
@@ -0,0 +1,115 @@
...
+              <TextField type="number" id="tax" label="Tax" variant="standard" fullWidth />
+            </Grid>
+
+            <Grid item xs={1.2}>
+              <TextField type="number" id="price" label="Price" variant="standard" fullWidth />
+            </Grid>
+
+            <Grid item container xs={1.2} alignItems={'center'} justifyContent={'center'}>
+              {/* Pytanie: Jak zrobić autoimport do ikon */}
+              <IconButton aria-label="delete">
```
Comment (DBulinski) [inline src/components/Invoice.tsx:111]: I think it should work out of the box, are you using Webstorm or VS Code?
Screenshot 2023-08-02 at 09 15 35

File: package.json (hunk 1 above)
Comment (broslukasz) [reply package.json]: No, I can replace it

File: package.json (hunk 2 above)
Comment (broslukasz) [reply package.json:47]: Il look for a solution in a spare time

File: src/App.tsx (hunk 3 above)
Comment (broslukasz) [reply src/App.tsx:12]: lets discuss about it. for me is kind of optional

File: src/App.tsx (hunk 4 above)
Comment (broslukasz) [reply src/App.tsx]: corrected

File: src/components/Invoice-list.tsx (hunk 5 above)
Comment (broslukasz) [reply src/components/Invoice-list.tsx]: What do you mean by valid label?

File: src/components/Invoice.tsx (hunk 7 above)
Comment (broslukasz) [reply src/components/Invoice.tsx]: corrected

File: src/App.tsx (hunk 8 above)
Comment (broslukasz) [reply src/App.tsx]: right, changed

File: src/components/Invoice.tsx (hunk 12 above)
Comment (broslukasz) [reply src/components/Invoice.tsx:111]: vs code

File: src/App.tsx (hunk 3 above)
Comment (broslukasz) [reply src/App.tsx:12]: changed

File: src/components/Invoice-list.tsx (hunk 6 above)
Comment (broslukasz) [reply src/components/Invoice-list.tsx]: some default imports not work e.g for adapter js.

File: src/components/Invoice-list.tsx (hunk 5 above)
Comment (DBulinski) [reply src/components/Invoice-list.tsx]: `Simple table` doesn't seem to be descriptive, should be probably more like a `Invoices table` or something like that.
